qa/L0_backend_python/lifecycle/lifecycle_test.py

```diff
@@ -241,6 +241,70 @@ def test_infer_pymodel_error(self):
             initial_metrics_value,
         )
 
+    def test_grpc_strict_error_on(self):
```

Comment:
Added some more tests to PR test plan description, just an FYI of some possibly impacted tests